### ISSUE TITLE
All routes not found on MacOS

### ISSUE THREAD

**Issue by vergoicincom** [jchristn/WatsonWebserver#59]
Hi, I'm make an app work fine on Windows. But on MacBook display "Not found (Not found)" any route.
Here is my code:
`
static HttpRequestService request;
        static void Main(string[] args)
        {
            Server s = new Server("127.0.0.1", 19000, false, DefaultRoute);

            s.Routes.Static.Add(WatsonWebserver.HttpMethod.GET, "/app.js", async (ctx) => {
                ctx.Response.ContentType = "application/javascript; charset=utf-8";
                await ctx.Response.Send(Properties.Resources.app);
            });
            s.Routes.Static.Add(WatsonWebserver.HttpMethod.GET, "/app.css", async (ctx) => {
                ctx.Response.ContentType = "text/css; charset=utf-8";
                await ctx.Response.Send(Properties.Resources.app1);
            });
            s.Routes.Static.Add(WatsonWebserver.HttpMethod.GET, "/logo.png", async (ctx) => {
                ctx.Response.ContentType = "image/png";
                await ctx.Response.Send(Properties.Resources.logo);
            });
            s.Routes.Static.Add(WatsonWebserver.HttpMethod.GET, "/login", AuthorizeAsync);
            s.Start();

            Console.WriteLine("Press ENTER to exit");
            Console.ReadLine();
        }
        private static async Task AuthorizeAsync(HttpContext ctx)
        {
            var login = new Login(request);
            try
            {
                var l = await login.GetLogin();
                ctx.Response.StatusCode = 200;
                ctx.Response.ContentType = "application/json; charset=utf-8";
                await ctx.Response.Send(l.ToJson());
                return;
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
        }
        static async Task DefaultRoute(HttpContext ctx)
        {
            ctx.Response.StatusCode = 200;
            ctx.Response.ContentType = "text/html; charset=utf-8";
            await ctx.Response.Send(Properties.Resources.index);
            return;
        }`

Thanks!

I'm using .Net 5.0

**Comment by jchristn** [comment]
Thanks @vergoicincom, looking into this.

**Comment by jchristn** [comment]
Hi @vergoicincom I just tested on Big Sur 11.1 using the ```Test.Default``` project and it worked for me:
![image](https://user-images.githubusercontent.com/1421081/112182078-35ef8680-8bba-11eb-97a8-f2b6dd44d010.png)

Here is the code change I made:
```csharp
            _Server.Routes.Static.Add(HttpMethod.GET, "/login", async (ctx) =>
            {
                ctx.Response.StatusCode = 200;
                ctx.Response.ContentType = "text/plain";
                await ctx.Response.Send("Login static route");
                return;
            });
```

Could you test in the ```Test.Default``` project and see if it works?

Also can you try changing your ```AuthorizeAsync``` method to ```internal``` and then ```public``` to see if this makes a difference?

**Comment by jchristn** [comment]
Hmmm, I just changed one of my routes to ```private``` and it didn't make a difference on 11.1...  Would you be able to reproduce in the ```Test.Default``` project?

**Comment by vergoicincom** [comment]
@jchristn Thanks you for supported!

My problem solved when I changed `127.0.0.1` to `localhost`. It is the difference on my code and `Test.Default`

Thanks you very much!

**Comment by jchristn** [comment]
Good to know!  Operating systems are sensitive to cases where the listener prefix (ie 127.0.0.1) is compared to the value in the incoming request's HOST header.
